feeds/indio/indio/files/etc/rc.local

```diff
@@ -0,0 +1,5 @@
+# Put your custom commands here that should be executed once
+# the system init finished. By default this file does nothing.
+/bin/sh /bin/hwwatchdog.sh &
+/bin/sh /bin/sys_led.sh &
+exit 0
```

Comment:
*nix has been using SYSV init for many decades. please do not re-invent it by using rc.local